changelog/issue-5666.md

```diff
@@ -0,0 +1,5 @@
+audience: worker-deployers
+level: patch
+reference: issue 5666
+---
+The generic-worker no longer panics if it gets no HTTP responses from Queue for over 15 minutes.
```

Comment:
where this number `15` is coming from?

Reply:
https://github.com/cenkalti/backoff/blob/v3/exponential.go#L79